actions/translation-cursor-deploy/main.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Custom Cursor Deploy GitHub action
+ * @see {@link https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action}
+ *
+ * Updates the deployment translation cursor file in the S3 bucket.
+ */
+
+import * as core from '@actions/core'
+import {exec} from '@actions/exec'
+
+import {writeLineToFile, runAction, fileExistsInS3} from '../utils'
+
+runAction(async () => {
+    const bucket = core.getInput('bucket_name', {required: true})
+    const hash = core.getInput('hash', {required: true})
+
+    await cursorDeploy({
+        bucket,
+        hash
+    })
+})
+
+const PREVIOUS_HASH_COMMAND = 'previous' // special command for rollback to hash from /previous
```

Comment:
This seems quite hacky to me (i.e. reusing hash to also double as a command sometimes). Maybe we can use the same concept of "deploy mode" we have in the regular deploy action, this would make things much more explicit.

Reply:
Could elaborate about concept of "deploy mode" ?